### PR TITLE
Removes processing machinery (profiling edition)

### PR DESCRIPTION
**Pull request by DominikPanic**
With (~2 minutes):
https://puu.sh/Daufo/cc94a49feb.png
Without (~20 seconds):
https://puu.sh/DauiT/11b64e5a70.png

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -155,8 +155,6 @@
 		update_icon()
 		addtimer(CALLBACK(src, .proc/update), 5)
 
-	start_processing()
```
Comment (spookydonut) [inline code/modules/power/apc.dm]: I'm assuming apcs work via powernet/area processing?

Comment (DominikPanic) [reply code/modules/power/apc.dm]: I'll ask Menshin

**Comment by spookydonut** [comment]
Whats the reduction in the number of processing machines?

**Comment by DominikPanic** [comment]
> 
> 
> Whats the reduction in the number of processing machines?

Around 2-3000

**Pull request merged**